examples/experimental/potree/src/potree-tile-3d-layer.ts

```diff
@@ -0,0 +1,79 @@
+import {Tile3DLayer, Tile3DLayerProps} from '@deck.gl/geo-layers';
+import {UpdateParameters} from '@deck.gl/core/typed';
+import { Viewport } from '@deck.gl/core';
+import {Source} from '@loaders.gl/loader-utils';
+import { PointcloudTileset } from './pointcloud-tileset';
+
+export type PotreeTile3DLayerProps = {
+  dataSource: Source
```

Comment:
I know the naming is confusing, but a `Source` is a descriptor object with a method that can create a `DataSource`, so let's try to keep the concepts separate...
```suggestion
  source: Source
```

Reply:
We could accept a pre-created dataSource, or take an array of Sources

Reply:
Done